docs/getting-started/local-setup/setting-up-a-local-chain.md

```diff
@@ -27,6 +27,17 @@ which is ready to accept transactions and interact with other Gno nodes.
 
 ![gnoland start](../../assets/getting-started/local-setup/setting-up-a-local-chain/gnoland-start.gif)
 
+:::info Lazy init
+
+Starting a Gno blockchain node using just the `gnoland start` command implies a few things:
```

Comment:
Please break lines at 80 chars for easier readability with raw markdown. Please do it in the whole file

Reply:
This is the default format break in my IDE

If you want to enforce a standard line break, please open up a PR that adds some script for formatting, ex `docs format`

Reply:
Please make sure this task is on the gno repo, and not on the private devrel repo 🙏 

If I applied this fix in this PR, I'd also need to apply it to many other documentation pages (see the majority of `concepts`), not the topic of this PR -- so I'll keep the formatting as is, and wait for your changes in the future 🙏